pom.xml

```diff
@@ -6,7 +6,7 @@
 
   <name>conductor</name>
   <url>http://github.com/conductor-framework/conductor</url>
-  <version>2.4.6</version>
+  <version>2.6.0</version>
```

Comment:
I think the next point release should be `2.5.0` not 6.

Reply:
I was going to bring it in line with our release tags. Last tag was `2.5.4` and the version in the pom trailed behind. https://jitpack.io/#willowtreeapps/conductor